pycroft/model/user.py

```diff
@@ -78,7 +78,8 @@ class User(ModelBase, UserMixin):
         # but between punctuation characters must be characters or numbers
         (?:[._-]?[a-z0-9])+$
         """, re.VERBOSE)
-    email_regex = re.compile(r"^[a-zA-Z0-9]+(?:(?:\+|-|_|\.)[a-zA-Z0-9]+)*"
+    email_regex = re.compile(r"^[a-zA-Z0-9!#$%&'*+\-/=?^_`{|}~]+"
```

Comment:
I'm not sure whether we should allow such email addresses. Wasn't it the case that this regex has been designed rather simple on purpose? For instance, email addresses with two subsequent underscores are invalid as well.
I  know the address you're referring to, and @lightning1 told me it was probably a mistake.